th2-service/templates/ingress.yaml

```diff
@@ -25,16 +26,16 @@ spec:
       - backend:
           serviceName: infra-editor
           servicePort: 8080
-        path: /editor/(.*)
+        path: /editor($|/.*)
       - backend:
           serviceName: infra-mgr
           servicePort: 8080
-        path: /editor/backend/(.*)
+        path: /editor/backend($|/.*)
```

Comment:
won't work for the second level endpoints, but it is not required for non-human urls, please remove